### PR TITLE
Column View improvements

### PR DESCRIPTION
**Pull request by TyJOrtiz**
**Resolved / Related Issues**
Itemize resolved / related issues by this PR.
- Related ... #4176 

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@TyJOrtiz Can you resolve the merge conflicts?

**Comment by TyJOrtiz** [comment]
> @TyJOrtiz Can you resolve the merge conflicts?

@yair done

**Comment by TyJOrtiz** [comment]
never mind, something's messed up... i hate resolving conflicts on the github site

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -1947,10 +1947,10 @@
   <data name="PinItemToStart.Text" xml:space="preserve">
     <value>Pin to the Start Menu</value>
   </data>
-    <data name="UnpinItemFromStart.Text" xml:space="preserve">
+  <data name="UnpinItemFromStart.Text" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:1950]:
```suggestion
    <data name="UnpinItemFromStart.Text" xml:space="preserve">
```

File: Files/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -1947,10 +1947,10 @@
...
     <value>Unpin from the Start Menu</value>
   </data>
-    <data name="BaseLayoutContextFlyoutColumn.Text" xml:space="preserve">
+  <data name="BaseLayoutContextFlyoutColumn.Text" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:1953]:
```suggestion
    <data name="BaseLayoutContextFlyoutColumn.Text" xml:space="preserve">
```

File: Files/UserControls/NavigationToolbar.xaml [hunk 3]
```diff
@@ -1109,6 +1109,21 @@
...
+                                        CommandParameter="{xh:SystemTypeToXaml Bool=True}"
+                                        CornerRadius="4"
+                                        IsChecked="{x:Bind LayoutModeInformation, Converter={StaticResource LayoutModeConverter}, Mode=OneWay, ConverterParameter=ColumnInfo}"
+                                        Style="{StaticResource ToggleButtonRevealStyle}"
+                                        ToolTipService.ToolTip="Column View">
+                                        <FontIcon
+                                            Margin="0,2,0,0"
+                                            FontSize="16"
+                                            Glyph="&#xE8C0;" />
+                                    </RadioButton>
```
Comment (yaichenbaum) [inline Files/UserControls/NavigationToolbar.xaml:1126]:
```suggestion
```

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**